ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.19.yaml

```diff
@@ -662,6 +662,22 @@ tests:
       TEST_TYPE: conformance-parallel
       USE_RAMFS: "true"
     workflow: openshift-e2e-libvirt-upi
+- as: ocp-e2e-ovn-remote-libvirt-ppc64le-s2s
+  capabilities:
+  - intranet
```

Comment:
We can replace these two lines with: `cluster: build10`
This will allow us to hopefully properly test the intranet capabilities *without* the conflicting IP space.